pulsar-client-cpp/lib/stats/ProducerStatsImpl.cc

```diff
@@ -41,52 +42,60 @@ std::string ProducerStatsImpl::latencyToString(const LatencyAccumulator& obj) {
     return os.str();
 }
 
+struct ProducerStatsImpl::AsyncCallbackLock
+{
+   std::mutex mutex;
+   bool enabled = true;
```

Comment:
The similar question as before. BTW, I think we can wrap it in another header file. It only requires including `<mutex>` header so it's not a big issue.

Reply:
Moved to AsyncCallbackLock.h